### PR TITLE
Fix some Tests for MOV/LEA comments removal

### PR DESCRIPTION
**Pull request by thestr4ng3r**

### CODE REVIEW

File: new/db/cmd/cmd_pdr [hunk 1]
```diff
@@ -420,8 +420,7 @@ EXPECT=<<RUN
 | 0x00005b5f      48d1fe         sar rsi, 1
 | 0x00005b62      7414           je 0x5b78
 | ----------- true: 0x00005b78  false: 0x00005b64
-| 0x00005b64      488b053dc401.  mov rax, qword [reloc._ITM_registerTMCloneTable] ; MOV rax = [0x21fa8] = 0x0 sym.imp.__sprintf_chk
```
Comment (radare) [inline new/db/cmd/cmd_pdr]: This sprintf chk was coming from..

Comment (thestr4ng3r) [reply new/db/cmd/cmd_pdr]: from wat

File: new/db/formats/mangling/bin [hunk 2]
```diff
@@ -83,8 +83,7 @@ RUN
 NAME=C++ demangle relocs
 FILE=../bins/elf/libstdc++.so.6
 EXPECT=<<CMDS
-            0x00091004      488b159d880f.  mov rdx, qword [reloc.vtableforstd::bad_alloc_168] ; MOV rdx = [0x1898a8] = 0x0 class.std::__use_cache_std::__numpunct_cache_wchar_t
-                                                                       ; [0x1898a8:8]=0
+            0x00091004      488b159d880f.  mov rdx, qword [reloc.vtableforstd::bad_alloc_168] ; [0x1898a8:8]=0
```
Comment (radare) [inline new/db/formats/mangling/bin]: we are missing some  numpunct reference here. is this  a false positive?

Comment (thestr4ng3r) [reply new/db/formats/mangling/bin]: The "class.std::__use_cache_std::__numpunct_cache_wchar_t" is part of the MOV comment from what I understand.

Comment (radare) [reply new/db/formats/mangling/bin]: yes, but. it is meaningful? because if so we are losing information here

Comment (thestr4ng3r) [reply new/db/formats/mangling/bin]: I have to check. If it is meaningful in any way, asm.refptr should probably print it then.

Comment (thestr4ng3r) [reply new/db/formats/mangling/bin]: Ok now I know the difference. asm.refptr does print these flags too in general:
Bildschirmfoto vom 2019-07-05 20-53-39
However this is explicitly disabled when the address is between -512 and 512. In the test it's 0 and `class.std::__use_cache_std::__numpunct_cache_wchar_t` just happens to be one of many flags at 0, so it's completely meaningless in this particular case.

**Comment by radare** [comment]
Ok lets go for merging then

**Pull request merged**